torch/distributions/transforms.py

```diff
@@ -167,6 +174,9 @@ def __init__(self, loc, scale, event_dim=0):
         self.scale = scale
         self.event_dim = event_dim
 
+    def __eq__(self, other):

```

Comment:
I'm pretty sure this is not correct. Should this return a bool matrix for batched `TransformedDistribution`s?

Reply:
No, `__eq__` should always return a Python boolean